tests/format_tests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+from pycpfcnpj import cpf, cnpj, mask
+
+
+class FORMATTests(unittest.TestCase):
+    """docstring for CPFTests"""
+
+    def setUp(self):
+        self.valid_cpf = '11144477735'
+        self.invalid_cpf = '11144477736'
+        self.invalid_cpf_str = '1234'
+        self.valid_cnpj = '11444777000161'
+        self.invalid_cnpj = '11444777000162'
+        self.invalid_cnpj_str = '1234'
+
+    def test_mask_valid_cpf(self):
+        self.assertTrue(cpf.validate(mask.mask_cpf_cnpj(self.valid_cpf)))
```

Comment:
You already know that's a valid cpf. I think you should test whether or not the masked was properly applied. This comment applies for the tests below, except for the invalids that should check the exception raised.